battle-service/src/main/java/com/company/battle/controllers/GameController.java

```diff
@@ -0,0 +1,43 @@
+package com.company.battle.controllers;
+
+import com.company.common.dtos.GameDto;
+import com.company.common.dtos.SettingsDto;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("api/v1/game")
+public class GameController {
+
+    @GetMapping("/{id}")
+    GameDto getGameById(@PathVariable UUID id) {
```

Comment:
package visible or public?

Reply:
I don't understand what you mean. Could you please explain?

Reply:
i mean access modifiers for controller's methods
https://www.mygreatlearning.com/blog/the-access-modifiers-in-java/#:~:text=non%2Daccess%20modifiers.-,What%20are%20Access%20Modifiers%3F,protected%2C%20default%2C%20and%20private.

any reasons you made it package visible(default)?

Reply:
Yes, we need public methods. I changed